tests/integration_test.rs

```diff
@@ -121,3 +122,73 @@ fn test_show_source() -> Result<()> {
     assert!(str::from_utf8(&output.get_output().stdout)?.contains("l = 1"));
     Ok(())
 }
+
+#[test]
+fn test_ruff_black_compatibility() -> Result<()> {
+    let fixtures_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("resources")
+        .join("test")
+        .join("fixtures");
+
+    let paths: Vec<walkdir::DirEntry> = WalkDir::new(fixtures_dir)
+        .into_iter()
+        .filter(|entry| {
+            entry.as_ref().map_or(true, |entry| {
+                let file_name = entry.path().to_string_lossy();
+                file_name.ends_with(".py")
```

Comment:
Stub files (`.pyi`) should also be checked.

Reply:
Thanks.